src/libs/actions/PersonalDetails.js

```diff
@@ -329,7 +329,12 @@ function setAvatar(file) {
                 return;
             }
 
-            setPersonalDetails({avatar: response.s3url, avatarUploading: false});
+            let {avatar, avatarThumbnail} = response;
+
+            // TODO: remove the following 2 lines once https://github.com/Expensify/Web-Expensify/pull/34469 is merged, also change above access to const
+            avatar = avatar || response.s3url;
+            avatarThumbnail = avatarThumbnail || response.s3url;
+            setPersonalDetails({avatar, avatarThumbnail, avatarUploading: false}, false);
```

Comment:
Since `User_UploadAvatar` currently still just returns `s3url` in the response, I've added a default here so that we don't have to worry about deploying this and https://github.com/Expensify/Web-Expensify/pull/34469 (which changes it to return `avatar` and `avatarThumbnail`) simultaneously.